lib/server.js

```diff
@@ -25,7 +25,23 @@ const status = (conn, params = {}) => {
   ).then(httpBody);
 };
 
+const properties = (conn, names = []) => {
+  const headers = conn.headers();
+  headers.set('Accept', 'application/json');
+  const queryString = names.map(name => `name=${name}`).join('&');
```

Comment:
should we use `encodeURIComponent` on `${name}` here?

Reply:
It's what the querystring package does, so it's probably good to be consistent. Good catch, thank you